### PR TITLE
20: new ros node with subscriber and publisher

### PR DESCRIPTION
**Pull request by kiran-mohan**
Deletes content of existing ino file, and replaces with a simple subscriber - publisher. 
The publisher simply publishes "I'm alive!" to the /heartbeat topic.
The subscriber subscribes to an std_msgs::Empty message on the /listener topic, and toggles the LED state.

### CODE REVIEW

File: kitt_teensy_ros_node/kitt_teensy_ros_node.ino [hunk 1]
```diff
@@ -14,729 +14,51 @@
...
-  ros_nh.advertise(ros_encoder_right_pub);
-  ros_nh.advertise(ros_estop_pub);
-  ros_nh.advertise(ros_teensy_debug_pub);
+/**
+ * @brief toggle LED state
+ * @param std_msgs::Empty an empty message
+ */
+void listenerCallback(const std_msgs::Empty& listener_msg)
+{
+  uint8_t toggled_led_state = (digitalRead(LED_PIN) == HIGH) ? LOW : HIGH;
```
Comment (griswaldbrooks) [inline kitt_teensy_ros_node/kitt_teensy_ros_node.ino]: Don't need extra parens.
I'd say using the ternary operator in this very very short case is fine, but anything longer than this I would ask to be refactored.

File: kitt_teensy_ros_node/kitt_teensy_ros_node.ino [hunk 2]
```diff
@@ -14,729 +14,51 @@
...
-  motor_left_speed = 0.0;
-  motor_right_speed = 0.0;
-  external_stop = false;
-  fob_mode = 0;
-  reference_wl = 0.0;
-  reference_wr = 0.0;
-  pose.x = 0.0;
-  pose.y = 0.0;
-  pose.theta = 0.0;
+  heartbeat_msg.data = "I'm alive!";
```
Comment (griswaldbrooks) [inline kitt_teensy_ros_node/kitt_teensy_ros_node.ino]: In the future, let's make this a timestamp we can initialize with the rest of the ros system.

Comment (illiu) [reply kitt_teensy_ros_node/kitt_teensy_ros_node.ino]: Yes

**Comment by griswaldbrooks** [comment]
Forgot to approve :(

**Pull request merged**